cmake/FindCoyoteHW.cmake

```diff
@@ -189,7 +189,7 @@ set(HCLK_F 450 CACHE STRING "HBM clock frequency")
 
 # Clock uncertainty for HLS synthesis; default 27% since HLS estimates can be different from the actual PnR
 # Therefore, HLS synthesis should always be performed conservatively, with a higher clock uncertainty
-set(HLS_CLOCK_UNCERTAINTY "27" CACHE STRING "HLS synthesis clock uncertainty [%]")
+set(HLS_CLOCK_UNCERTAINTY "0" CACHE STRING "HLS synthesis clock uncertainty [%]")
```

Comment:
We should revert this change; I don't think it applies to the TCP HLS stack. In generally, HLS synthesis should have some clock uncertainty so that it's more conservative during the HLS scheduling for later PnR.